backend/controllers/minify.js

```diff
@@ -41,9 +41,34 @@ module.exports.findUrlById = async (req, res) => {
       })
 }
 
+module.exports.getAliasStatus = async (req, res) => {
+   try{
+      const alias = req.body.alias;
+      const aliasStatus = await minifed_urls.findOne({ alias });
+
+      if(!aliasStatus){       
+         return res.status(200).json("success: true");
+      }else{
+         return res.status(400).json("success: false")
+      }
+      }catch (error){
+         console.error(error)
+      } 
+}
+
 module.exports.addURL = async (req, res) => {
    const alias = nanoid(5)
    const minifiedUrl = base_url + alias
+   
+   try{
+   const aliasPresent = await minifed_urls.findOne({ alias });
+   if(aliasPresent){      
+      return res.status(400).json("success: false");
```

Comment:
Return  like this .json({success: false})

Reply:
done